config/rbac/role.yaml

```diff
@@ -28,7 +28,7 @@ rules:
   verbs:
   - get
   - list
-  - watch
+  - watch // Also used by promiseRelease
```

Comment:
I _think_ this is what caused issues on another commit when it got to the helm chart. Let's review before merging.